code/game/objects/items/weapons/implant/implants/carrion/holo_spider.dm

```diff
@@ -0,0 +1,233 @@
+/obj/item/implant/carrion_spider/holographic
+	name = "holographic spider"
+	desc = "A spider with a peculiarly reflective surface"
+	icon_state = "spiderling_holographic"
+	slot_flags = SLOT_ID | SLOT_BELT | SLOT_EARS | SLOT_HOLSTER | SLOT_BACK | SLOT_MASK | SLOT_GLOVES | SLOT_HEAD | SLOT_OCLOTHING | SLOT_ICLOTHING | SLOT_FEET | SLOT_EYES
```

Comment:
Is there a flag that includes all of these?

Reply:
I couldn't find one, so probably not.